### DISCUSSION TITLE
Can't see dates from Google Calender with only Busy state

### DISCUSSION THREAD

**Discussion by georgkrause** [bitfireAT/icsx5#26] [General]
I have a Google Calender, with a set of dates. These dates hide all details, its basically only exposing the start time and the end time and the event is called "busy". Sadly, non of these events are displayed and ICSx5 always tells me the calendar don't have any event.

An example event from the ICS looks like this:

```
BEGIN:VEVENT
DTSTART:20221104T093000Z
DTEND:20221104T100000Z
DTSTAMP:20220307T154542Z
UID:****
ATTENDEE;X-NUM-GUESTS=0:mailto:***
RECURRENCE-ID:20221104T093000Z
SUMMARY:Busy
END:VEVENT
BEGIN:VEVENT
DTSTART:20221028T093000Z
DTEND:20221028T100000Z
DTSTAMP:20220307T154542Z
UID:*******
ATTENDEE;X-NUM-GUESTS=0:mailto:***
RECURRENCE-ID:20221028T093000Z
SUMMARY:Busy
END:VEVENT
```

It works if I share all the information publicly.

**Comment by georgkrause** [comment]
A date which works looks like this:

```
BEGIN:VEVENT
DTSTART:20220307T170000Z
DTEND:20220307T180000Z
DTSTAMP:20220307T155311Z
UID:2rp6vvi53p6b1v0ma09eibtfd0@google.com
CREATED:20220307T155159Z
DESCRIPTION:
LAST-MODIFIED:20220307T155159Z
LOCATION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:testtermin
TRANSP:OPAQUE
END:VEVENT
```

**Comment by rfc2822** [comment]
Hi,

Basically the same as https://github.com/bitfireAT/davx5/issues/58 (non-recurring events with `RECURRENCE-ID`). We still have to find a solution for that.

**Comment by rfc2822** [comment]
See also #27

**Comment by devvv4ever** [comment]
@georgkrause Is it possible to send you an apk with a possible fix for this so that you can test it? In this case send an email to the email address that you can find at the top of the @bitfireAT page.

**Comment by georgkrause** [comment]
I got an APK for testing and this works like charm! Will there be a bugfix release shipping the changes?

**Reply by devvv4ever** [reply]
It should be already available in ICSx⁵ 2.0.2!
